content/influxdb/v2.0/upgrade/v1-to-v2.md

```diff
@@ -12,33 +12,180 @@ aliases:
+you may want to research using Telegraf instead.
+InfluxDB 2.0 doesn't support these protocols directly.
+Use Telegraf to translate from the source protocol to InfluxDB 2.0.
+
+### Kapacitor
+
+You can continue to use Kapacitor with InfluxDB OSS 2.0 under the following scenarios:
+
+1. Kapacitor Batch-style TICKscripts work with the 1.x read compatible API
+   Existing Kapacitor user credentials should continue to work using the [1.x compatibility API](/influxdb/v2.0/reference/api/influxdb-1x/).
+2. Kapacitor Stream-style TICKscripts will not work using the [subscription API]((/influxdb/v1.8/administration/subscription-management/)).
+   (There is no subscriptions API in InfluxDB 2.0.)
+   We recommend writing data directly to *both* Kapacitor and InfluxDB.
+   This allows stream tasks to continue to work.
+   To do this, configure two [InfluxDB output plugins](/telegraf/v1.16/plugins/#influxdb): one for Kapacitor and one for InfluxDB 2.0.
+   If you use other mechanisms to feed data into InfluxDB, Telegraf can be used as an intermediate layer to feed both Kapacitor and InfluxDB.
+
+   Other architectures and mechanisms possible, too.
+   Need more help? Reach out in the [community forums](https://community.influxdata.com).
+
+### User migrations
+
+`influxd upgrade` migrates existing 1.x users and their permissions.
+However, *administrative users are not migrated*.
+
+With the InfluxDB 1.x CLI, perform a `show users` query.
+Any users labeled "admin" will *not* be migrated.
+
+If using an admin user for visualization or Chronograf administrative functions, you may want to create a new read only user before upgrading.
+Admin rights are granted to the primary user created in the 2.0 setup process via the `influxd upgrade` command.
+(This provides you with the opportunity to re-assess who should be granted admin level access as a part of your InfluxDB 2.0 setup.)
+
+### Dashboards
+
+You can continue to use your existing dashboard/visualization tools with InfluxDB 2.0 via the [1.x read compatibility API](/influxdb/v2.0/reference/api/influxdb-1x/).
+[DBRP mappings](/influxdb/v2.0/reference/api/influxdb-1x/dbrp/) are created as part of the upgrade process to ensure existing users can execute InfluxQL queries with the appropriate permissions.
+
+However, if your dashboard tool is configured using a user with admin permissions,
+you will need to create a new read-only user with the appropriate database permissions *before* upgrading.
+This new username/password combination should be used within the data source configurations to continue to provide read-only access to the underlying data.
+
+Ensure your dashboards are all functioning before upgrading.
+
+### Other data
 
-{{% note %}}
 The 1.x `_internal` database is not migrated with the `influxd upgrade` command.
+
+### Secure by default
+
+Running InfluxDB with the setting
```

Comment:
If this isn't inline you should introduce it with a colon and follow it with a complete sentence.